packages/one-app-dev-bundler/esbuild/utils/load-styles.js

```diff
@@ -59,7 +59,8 @@ const loadStyles = async ({
   let cssModulesJSON = {};
   const purgecssConfig = getModulesBundlerConfig('purgecss') || {};
   const result = await postcss([
-    (process.env.NODE_ENV === 'production' && !purgecssConfig.disabled) && purgecss(purgecssConfig),
+    // purgecss will not execute unless `disabled is set to `false` (opt-in by default)
+    purgecssConfig.disabled === false && purgecss(purgecssConfig),
```

Comment:
to go with the other comment we could do something like:
```js
if(purgecssConfig.hasOwnProperty('disabled')) console.warn("purge css disabled flag is deprecated. please use 'enabled'")
const enablePurgecss = purgecssConfig.enabled || (purgecssConfig.disabled === false)
const result = await postcss([
   enablePurgecss && purgecss(purgecssConfig),
```

Reply:
That's quite nice - I'm happy with that, can add that in with the necessary config changes

Reply:
So this literally works, but I can't seem to get the console warning to appear when bundling a test module with `disabled` set to `true` or `false`. Is console output hidden by default, or have I done something wrong here?

Reply:
This is more a bug in the logging of the bundler itself, which is being addressed soon.

The log should be put out at this level 😊